backend/src/main/java/quarano/actions/web/ActionRepresentations.java

```diff
@@ -186,17 +211,33 @@ public DailyItems(LocalDate date, List<ActionItem> items, MessageSourceAccessor
 		}
 	}
 
-	static class CaseActionSummary {
+	@Relation(collectionRelation = "actions")
+	static class CaseActionSummary extends RepresentationModel<CaseActionSummary> {
 
 		private final ActionItems items;
 		private final TrackedCaseSummary summary;
-		private final TrackedCase trackedCase;
+		private final @Getter(onMethod = @__(@JsonIgnore)) TrackedCase trackedCase;
 
 		CaseActionSummary(TrackedCase trackedCase, ActionItems items, TrackedCaseSummary summary) {
 
 			this.trackedCase = trackedCase;
 			this.items = items;
 			this.summary = summary;
+
+			add(getDefaultLinks(trackedCase));
+		}
+
+		@SuppressWarnings("null")
+		public static Links getDefaultLinks(TrackedCase trackedCase) {
+
+			var caseId = trackedCase.getId();
+			var actionController = on(ActionItemController.class);
+			var caseController = on(TrackedCaseController.class);
+
+			return Links.of(MvcLink.of(actionController.allActions(caseId, null), IanaLinkRelations.SELF))
```

Comment:
You find that more readable? I don't, because stream processing and link assembling are too close together.

`
return trackedCase.getOriginCases().stream().map(it -> MvcLink.of(caseController.getCase(it.getId(), it.getDepartment()), ORIGIN_CASES))
                                      .collect(Links.collector())
                                      .and(MvcLink.of(actionController.allActions(caseId, null), IanaLinkRelations.SELF));`

I think the following is okay.
`			var links = trackedCase.getOriginCases().stream()
					.map(it -> MvcLink.of(caseController.getCase(it.getId(), it.getDepartment()), ORIGIN_CASES))
					.collect(Links.collector());
			return links.and(MvcLink.of(actionController.allActions(caseId, null), IanaLinkRelations.SELF));`

Reply:
Why does the codeblock reformat the code? 🤷

Reply:
This looks pretty decent to me?

```java
return trackedCase.getOriginCases().stream()
  .map(it -> MvcLink.of(caseController.getCase(it.getId(), it.getDepartment()), ORIGIN_CASES)) 
  .collect(Links.collector()) // produces Links
  .and(MvcLink.of(actionController.allActions(caseId, null), IanaLinkRelations.SELF));
```